.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,29 @@
+name: Docuemntation
```

Comment:
```suggestion
name: Documentation
```

Reply:
I don't think I ever would have noticed this :)

Reply:
ah! Thank you!!